### PR TITLE
Made corrections so reads as distributed MESA packages

### PR DESCRIPTION
**Pull request by tpike3**
Made the requested changes so it reads as shared packages from different github repositories instead of a consolidated repository. 

The one challenge I did see was where links to different packages can be stored. I linked to the MESA wiki main page, but at the least probably needs to be a subpage, or other dedicated location. 

Let me know if this meets the mail.

### CODE REVIEW

**Comment by tpike3** [comment]
Apologies, found 2 grammatical mistakes.

**Comment by jackiekazil** [comment]
Merging into the branch and reviewing tonight.

File: docs/packages/package_main.rst [hunk 1]
```diff
@@ -3,46 +3,26 @@
...
+		Volker Grimm et al, "Pattern Oriented Modeling of Agent Based Complex Systems: Lessons from Ecology" Science, New Series, Vol. 310, No. 5750
+		(Nov. 11, 2005), pp. 987-991
 
-	Volker Grimm et al, "Pattern Oriented Modeling of Agent Based Complex Systems: Lessons from Ecology" Science, New Series, Vol. 310, No. 5750
-	(Nov. 11, 2005), pp. 987-991
 
 
-Mesa Packages Purpose: 
-----------------------
+The Mesa core functionality is just a subset of what we believe researchers creating Agent Based Models (ABMs) will use. We designed Mesa to be extensible, so that individuals from various domains can build and maintain their own Mesa packages in pursuit of "unifying algorithmic theories of the relation between adaptive behavior and system complexity."
```
Comment (jackiekazil) [inline docs/packages/package_main.rst]: @tpike3 ... re: "“unifying algorithmic theories of the relation between adaptive behavior and system complexity.”" ... why is that in quotes?

File: docs/packages/package_main.rst [hunk 2]
```diff
@@ -3,46 +3,26 @@
...
 
+The purpose of this guide is to support new programmers by providing detailed guides on "How to" build and share a package. Let's get started!
 
-**MESA Packages is a repository to share packages for use in Agent Based Models (ABMs) in pursuit of "unifying algorithmic theories of the relation between adaptive behavior and system complexity."**
+	You want create and share your package but have never done it before -- start here: :ref:`package-development`
```
Comment (jackiekazil) [inline docs/packages/package_main.rst]: Lines 20-22 are a lot of words that don't need to be there. 
How about... 

Choose your adventure: 
* Learn how to create and share your package
* Learn how to incorporate a package from a Mesa user into your model

File: docs/packages/package_main.rst [hunk 3]
```diff
@@ -3,46 +3,26 @@
...
 
-	**(1) Provide code which can be integrated into MESA ABMs to enable greater exploration of complex phenomenon.**
-	
-	**(2) Support new programmers by providing detailed guides on "How to" build and share a package.**
+	You want to incorporate a package from a MESA user but don't know how--- try this: :ref:`user-guide`
```
Comment (jackiekazil) [inline docs/packages/package_main.rst]: MESA should be Mesa

File: docs/packages/Package Development/package development main.rst [hunk 4]
```diff
@@ -3,20 +3,18 @@
 Package Development: A "How-to Guide" for Developing Packages for Others to Use
 ================================================================================
 
-The purpose of this page is to get your mesa package sharing as quickly as possible. The authoritative guide for python package development is through the `Python 
-Packaging User Guide <https://packaging.python.org/>`_. This will take you through the entire process necessary for getting you package on the Python Package Index.
+The purpose of this page is to get your mesa package sharing as quickly as possible.
```
Comment (jackiekazil) [inline docs/packages/Package Development/package development main.rst]: Change to: The purpose of this page is help you set up and distribute your mesa package as quickly as possible.

File: docs/packages/Package Development/package development main.rst [hunk 5]
```diff
@@ -3,20 +3,18 @@
 Package Development: A "How-to Guide" for Developing Packages for Others to Use
 ================================================================================
 
-The purpose of this page is to get your mesa package sharing as quickly as possible. The authoritative guide for python package development is through the `Python 
-Packaging User Guide <https://packaging.python.org/>`_. This will take you through the entire process necessary for getting you package on the Python Package Index.
+The purpose of this page is to get your mesa package sharing as quickly as possible. 
 
 This "How-to Guide" uses GitHub to walk you through the process. However, other repositories will be able to provide similar services.
```
Comment (jackiekazil) [inline docs/packages/Package Development/package development main.rst]: What do you mean by "However, other repositories will be able to provide similar services." ... can you clarify? Do you mean other service providers (ie GitLab)?

File: docs/packages/Package Development/package development main.rst [hunk 6]
```diff
@@ -3,20 +3,18 @@
...
 
-
-Package Development Checklist (basic): Sharing your modules in seven steps
+Package Development Checklist (basic): Sharing your package in seven steps
```
Comment (jackiekazil) [inline docs/packages/Package Development/package development main.rst]: If this is "basic" ... where is advanced? If there is no advanced yet, then we should delete "basic".

**Comment by jackiekazil** [comment]
When you get done making your next set updates, I am going to add a few things.... 
- [ ] Add explainer at the top about setting up packages
-  [ ] Add a little more language to encourage the "right" license

File: docs/packages/Package Development/package development main.rst [hunk 7]
```diff
@@ -49,11 +47,14 @@ We recommend `Hitchhiker's Guide to Python <http://docs.python-guide.org/en/late
...
       Don't forgot to follow a good `structure <http://docs.python-guide.org/en/latest/writing/structure/>`_
 
-**7. Add a page to Mesa Packages WIKI page**
+**7. Let people know about your package on the MESA wiki page**
 
-      - `Build your package page <https://github.com/projectmesa/mesa/wiki>`_  
-      - Edit the Mesa Packages wiki page to add your package name
+      - `MESA Wiki Page <https://github.com/projectmesa/mesa/wiki>`_  
    
+You want to do even more. The authoritative guide for python package development is through the `Python Packaging User Guide <https://packaging.python.org/>`_. This will take you through the entire process necessary for getting your package on the Python Package Index.
```
Comment (jackiekazil) [inline docs/packages/Package Development/package development main.rst]: This is great! Maybe add a little more about why this is different than just putting on the wiki.

**Pull request merged**